modules/cnf-du-updating-crio.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-crio_{context}"]
+
+= Updating CRI-O
+
+In support of this feature, CRI-O supports new configuration settings. These are applied by the MachineConfig specified above.
+
+----
```

Comment:
* Please add "[source,terminal]" according to the doc guidelines for [code blocks](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).
* Consider a block title like ".Example CRI-O configuration for workload partitioning"